scripts/check_frontend_coverage.py

```diff
@@ -317,6 +317,7 @@
     'state-responses.directive.ts',
     'state-solution-editor.directive.ts',
     'state-top-answers-stats.service.ts',
+    'state-tutorial-first-time-backend-api.service.ts',
```

Comment:
We don't add new files to the list? Can you cover this file completely?

Reply:
Done